src/Algebra/Graph/Acyclic/AdjacencyMap.hs

```diff
@@ -504,3 +504,26 @@ induceEAM p m = es m `AM.overlay` vs m
   where
     es = AM.edges . filter p . AM.edgeList
     vs = AM.vertices . AM.vertexList
+
+-- | Constructs an acyclic graph from any graph based on
+-- a order of vertices to produce an acyclic graph.
```

Comment:
English is not my mother tongue, but maybe there is a typo here: "a order" -> "an order".